config/config.go

```diff
@@ -32,28 +36,43 @@ const (
 	ErrSchema errors.Kind = "config has an invalid schema"
 )
 
-// TryLoadRootConfig try to load the Terramate root config. It looks for the
+// Tree is the config tree.
+type Tree struct {
+	// Root is the configuration of this tree node.
+	Root hcl.Config
```

Comment:
Maybe call it Node instead of Root, but not sure, feel free to disregard (just tossing the idea).